uikit/viewmodels/src/main/java/com/sygic/maps/uikit/viewmodels/common/sdk/holders/NaviSignInfoHolder.kt

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) 2019 Sygic a.s. All rights reserved.
+ *
+ * This project is licensed under the MIT License.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+package com.sygic.maps.uikit.viewmodels.common.sdk.holders
+
+import com.sygic.sdk.navigation.warnings.NaviSignInfo
+
+data class NaviSignInfoHolder(val naviSignInfo: NaviSignInfo? = null) {
+
+    companion object {
+
+        @JvmStatic
+        fun empty(): NaviSignInfoHolder {
```

Comment:
nechcel si to skor mat val? toto skonstruuje novy objekt pri kazdom volani

Reply:
nejsem si jistý, zda tímto reálně něco "ušetříme", ale oki, upravím na val.

Reply:
no je to dost rozdiel, ci pri kazdom volani vytvoris novy objekt, alebo nie, ci preco myslis?